{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -42,8 +42,35 @@ def get_tm_session(session_factory, transaction_manager):
           with transaction.manager:
               dbsession = get_tm_session(session_factory, transaction.manager)
 
+    This function may be invoked with a ``request`` kwarg, such as when invoked
+    by the reified ``request.dbsession`` Pyramid Request attribute. The default
+    value, for backwards compatibility, is ``None``.
+
+    - The ``request`` kwarg is used to populate the SQLAlchmey Session's "info"
```

Comment:
Why is this a list item and not just a plain old paragraph?

Reply:
Also why is Session capitalized? It should either be lowercased to be a generic term of "session" or double-backticked as an inline literal.

Reply:
1. I will drop to regular paragraph. 
2. Within SQLAlchemy, "Session" is the class/concept name, while "session" is the abstract idea and usually an instance name; just like Pyramid has Request and "request".  I will change this to lowercase to keep in line with Pyramid docs.